### PR TITLE
Only include CSS that's required for error pages

### LINKED ISSUE #130: 404 and 500 error pages load entirety of GOV.UK Frontend / Design System CSS
The static error pages use a downloaded version of the GOV.UK Design System CSS - this should be replaced with only the styles needed for the page, which could be inlined.

### PR DESCRIPTION
**Pull request by injms**
## What 

The error pages were including all of the CSS from the GOV.UK Frontend design system. This has been changed to only include the styles needed for the pages. This reduced CSS from 87.4KB to 24.1KB.

The CSS filename has a hash of the file added to it to avoid a cache clash if it needed updating.

Fixes #130.

## Why

To make the (hopefully rarely seen) error pages faster to load.

## Visual differences

None.

## Local testing

Set `config.consider_all_requests_local` to `true` in `config/environments/development.rb` to see the errors when running the app locally.

### CODE REVIEW

File: public/404.html
Comment (alex-ju) [inline public/404.html:11]: XHTML ftw

**Review by alex-ju — APPROVED** [review]
Nice one, I'll use it in the other service too. Thank you for your investment!

**Comment by theseanything** [comment]
Looks good to me! We're going to have to hold off merging until we fix our deployment pipeline :(

File: public/404.html
Comment (injms) [reply public/404.html:11]: You knows it.

**Comment by injms** [comment]
> Looks good to me! We're going to have to hold off merging until we fix our deployment pipeline :(

Ah well - can you approve when it's okay to merge this in?

**Comment by injms** [comment]
Unblocked because #208 and #219 have been merged in.

**Pull request merged**
